firmware/application/apps/ui_looking_glass_app.cpp

```diff
@@ -471,9 +471,9 @@ GlassView::GlassView(
 
     button_marker.on_select = [this](ButtonWithEncoder&) {
         receiver_model.set_target_frequency(marker);  // Center tune rx in marker freq.
-        receiver_model.set_frequency_step(MHZ_DIV);   // Preset a 1 MHz frequency step into RX -> AUDIO
```

Comment:
This is how an app can pass override settings. The override will be applied after both defaults and app_settings.
NB: This does mean that app settings will end up saving these override values on exit.